### ISSUE TITLE
Gun acu is owerpowered against auroras

### ISSUE THREAD

**Issue by Ithilis** [FAForever/nomads#500]
not even that single shot kill them, that is 33% more efficient as other aucus aoe effect on first gun upgrade kill 2-3 auroras near as well, what easily can turn into masacre. Not sure about solution, but it is issue. 

Maybe, jast maybe we should go back to 100dmg per shot format :/

**Comment by Apofenas** [comment]
Hey! you were the one who suggested it in a first place to nerf double gun!

**Comment by Ithilis** [comment]
Yes i know, because it was too strong :). Now its too strong only against aeon :D, what is something..  but still im not totaly sure what to do with it. also there is capacitator that can interact, its kinda complicated : /
